static/app/routes.tsx

```diff
@@ -1889,20 +1889,22 @@ function buildRoutes() {
         componentPromise={() => import('sentry/views/profiling/content')}
         component={SafeLazyLoad}
       />
+
```

Comment:
This is kinda a personal nit, but I was trying to avoid having any blank line in route groups so it's easy to jump between 'paragraphs' of code, where each paragraph is a route group.